pkg/router/template/router.go

```diff
@@ -375,17 +372,6 @@ func (r *templateRouter) writeDefaultCert() error {
 	return nil
 }
 
-func (r *templateRouter) readState() error {
-	data, err := ioutil.ReadFile(filepath.Join(r.dir, routeFile))
```

Comment:
Delete the declaration of `routeFile` too.  (We can get this in a follow-up.)

Reply:
Removed